### DISCUSSION TITLE
CSV Parsing feature request - Skip Trailer Record(s)

### DISCUSSION THREAD

**Discussion by chrisfw** [duckdb/duckdb#4157] [Ideas]
We deal with a lot of CSV files that contain a Trailer record as an EOF marker.  It would be great to be able to specify to the Duckdb CSV parser a number of records to skip at the end of a CSV file just as it exists for the beginning of the file.

**Comment by madhavanhere** [comment]
The only option right now appears to be to set null_padding=true with a where clause to weed out trailers. 
A 'default' skip trailer record would be very beneficial!

**Reply by jimdickson007** [reply]
I vaguely recall this was requested before and the response was it was hard because of parallelism etc
